app/src/main/java/org/openobservatory/ooniprobe/activity/MainActivity.java

```diff
@@ -26,6 +26,7 @@
     private static final String RES_ITEM = "resItem";
     private static final String MANUAL_UPLOAD_DIALOG = "manual_upload";
     private static final String ANALYTICS_DIALOG = "analytics";
+    private AtomicBoolean isTestRunning;
```

Comment:
I must say I am super confused by a boolean and a function with the same name. Also, we probably don't care about having a boolean and we can invoke the function is all cases.

Reply:
renamed in isUITestRunning.

The code was found in a fastlane QA